README.md

```diff
@@ -112,6 +112,8 @@ When `SIDEKIQ_ENABLED` is set we'll:
 The following ENV are available:
 
 - `SIDEKIQ_ENABLED`
+- `SIDEKIQ_QUEUES` - comma-separated custom queue names; if not specified, default queues are processed which are defined [here](./lib/roo_on_rails/sidekiq/settings.rb)
+- `SIDEKIQ_PERMITTED_LATENCY_VALUES` - our default queues come with a permitted latency value which is used in monitoring and Hirefire scaling. For your custom queues, you will need to specify the permitted latency in this format: `QUEUE_NAME:VALUE:UNIT` e.g. `queue-one:1:day,queue-two:3:minutes,queue-three:2:seconds`
```

Comment:
It seems strange and error prone that there are two environment variables that need to match up for things to work, if they don't obey the `withinXunits` pattern. I'd suggest that we just use `SIDEKIQ_QUEUES` and have the latency as an optional setting, eg.

```
SIDEKIQ_QUEUES=within1hour,custom1:1day,within10minutes,custom2:3hours
```

i.e. the `:latency` is an optional part of the queue specification. I find the `1:day` syntax somewhat odd as it looks like a 3-tuple so I'd suggest either splitting on digits/non-digits as above, or something like `1.day` to make it clear it's a 2-tuple of `name:latency`.